docs/content_management/taxonomy/taxonomy_api.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Using the PHP API you can browse taxonomy entries, get their information and manage them.
+---
+
+# Taxonomy API
+
+To manage taxonomies, use `Ibexa\Contracts\Taxonomy\Service\TaxonomyServiceInterface`.
+
+## Getting taxonomy entries
+
+To get a single taxonomy entry, you can use `TaxonomyServiceInterface::loadEntryById()`
+and provide it with the numerical entry ID, or pass entry identifier and use `TaxonomyServiceInterface::loadEntryByIdentifier()`:
+
+``` php
+[[= include_file('code_samples/api/public_php_api/src/Command/TaxonomyCommand.php', 43, 46) =]]
+```
+
+You can also get a taxonomy entry from the ID of its underlying Content item, by using `TaxonomyServiceInterface::loadEntryByContentId()`.
+
+To get the root (main) entry of a given taxonomy, use `TaxonomyServiceInterface::loadRootEntry()`
+and provide it with the taxonomy name.
+
+To get all entries in a taxonomy, use `TaxonomyServiceInterface::loadAllEntries()`, provide it with the taxonomy identifier,
```

Comment:
Limiting the results is not really optional because it has limit of 30 by default.